docs/src/components/CodeLinesFromFile.tsx

```diff
@@ -0,0 +1,14 @@
+import CodeBlock, {Props} from '@theme/CodeBlock';
```

Comment:
this theme/package is included by default?

Reply:
Yes